assets/icons/stumbleupon.svg

```diff
@@ -6,5 +6,5 @@
     <mask id="mask-2" sketch:name="Clip 2">
         <use xlink:href="#path-1"></use>
     </mask>
```

Comment:
@mattolson  Do I need to also remove this tag? I only removed the attribute from the path tag below.

Reply:
No I think this one is fine.

Reply:
Yes, you need to remove `sketch:name` from the mask.

Reply:
You also need to remove the `xlink` prefix from the `use` tag: `<use href=...`